packages/shorebird_cli/lib/src/commands/patch/android_patcher.dart

```diff
@@ -79,7 +79,8 @@ class AndroidPatcher extends Patcher {
       aabFile = await artifactBuilder.buildAppBundle(
         flavor: flavor,
         target: target,
-        args: argResults.forwardedArgs,
+        args: argResults.forwardedArgs +
+            buildNameAndNumberArgsFromReleaseVersion(releaseVersion),
```

Comment:
where does `releaseVersion` come from now? It's not clear from the diff.

Reply:
It's provided to `buildPatchArtifact` by `PatchCommand`, which will use one of:

1. The `--release-version` arg
2. The release version selected by the user via `logger.chooseOne`
3. The version of the compiled app (CI only)